packages/shared/src/components/RichTextEditor/plugins/ButtonPlugin/components/ButtonToolbarButton.tsx

```diff
@@ -0,0 +1,65 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import {
+    ToolbarButton,
+    focusEditor,
+    isRangeInSameBlock,
+    someNode,
+    useEventPlateId,
+    usePlateEditorState,
+} from '@udecode/plate';
+import { triggerFloatingButton } from '../utils';
+import { BlockToolbarButtonProps } from '@udecode/plate';
+import { getHotkeyByPlatform, getTooltip } from '@frontify/fondue';
+
+const buttonRootClassNames =
+    'tw-ml-0.5 !tw-text-text-weak hover:!tw-bg-box-selected hover:!tw-text-box-selected-inverse hover:tw-rounded';
+
+const getButtonClassNames = (isEnabled = true) => ({
```

Comment:
could we just import this function from fondue?

Reply:
Done 👍🏻